lib/formulator.ex

```diff
@@ -26,13 +26,23 @@ defmodule Formulator do
       #=> <input id="user_name" name="user[email]" aria-label="email" type="text" value="">
 
   Passing other options:
-      <%= input form, :email, label: [class: "control-label"] %>
+      <%= input form, :email, as: :email, label: [class: "control-label"] %>
       #=> <label class="control-label" for="user_email">Email</label>
-      #=> <input id="user_email" name="user[email]" type="text" value="">
+      #=> <input id="user_email" type="email" name="user[email]" value="">
 
-      <%= input form, :email, class: "my-email-class", label: [class: "my-email-label-class"] %>
+      <%= input form, :email, as: :email, placeholder: "your@email.com", class: "my-email-class", label: [class: "my-email-label-class"] %>
       #=> <label class="my-email-label-class" for="user_email">Email</label>
-      #=> <input id="user_email" name="user[email]" type="text" value="" class="my-email-class">
+      #=> <input id="user_email" type="email" name="user[email]" placeholder: "your@email.com" value="" class="my-email-class">
+
+  With validation attributes:
```

Comment:
Seems like we need more explanation of what is happening exactly. From the outside it's not clear that this requires a changeset that has an applicable validation.